### DISCUSSION TITLE
Curious about OpenRPC Support

### DISCUSSION THREAD

**Discussion by zcstarr** [modelcontextprotocol/modelcontextprotocol#856] [Ideas - General]
Not sure if this is the right place for this, but I was checking out your spec and noticed a lot of overlap with [OpenRPC](https://open-rpc.org/)— think OpenAPI for JSON-RPC APIs. 

It could be awesome to collaborate on adding support for it in a minimally invasive way. Curious to hear your thoughts!

**Comment by jspahrsummers** [comment]
Supporting OpenRPC seems like it could be useful, assuming there's tooling out there to make use of it! I imagine it'd be pretty straightforward to generate an OpenRPC spec from the protocol.

I guess the key question is whether this will be concretely useful for integrations somehow. Would love to hear if that's the case (and how)!

**Comment by localden** [comment]
We've limited Discussions in this repository to meeting notes and are closing this thread. To continue the conversation, pick a channel from [Contributor Communication](https://modelcontextprotocol.io/community/communication). Thanks for posting.
